modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/SidebarPanel.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {fetch} from 'frontend-js-web';
+import React, {useImperativeHandle, useEffect, useState} from 'react';
+
+import Sidebar from './components/Sidebar';
+
+const noop = () => {};
+
+const SidebarPanel = React.forwardRef(
+	({fetchURL, onClose = noop, resourcePrimKey}, ref) => {
+		const [isOpen, setIsOpen] = useState(true);
+		const [resourceData, setResourceData] = useState();
+
+		const getData = (fetchURL, resourcePrimKey) => {
+			if (resourcePrimKey !== resourceData?.resourcePrimKey) {
+				const formData = new FormData();
+
+				formData.append(`resourcePrimKey`, resourcePrimKey);
+
+				fetch(fetchURL, {
```

Comment:
Since this is on a per-row basis... maybe it would be easier to just get the full url formed from the backend as a prop

Reply:
yep, I already change it :)

Reply:
![yourebreathtaking](https://user-images.githubusercontent.com/905006/87796777-1b565800-c84a-11ea-8d5f-acdf4e41703d.gif)